packages/mermaid/src/diagrams/sankey/parser/sankey.jison

```diff
@@ -11,14 +11,14 @@
 %lex
 
 %options case-insensitive
-%options easy_keword_rules
+%options easy_keyword_rules
```

Comment:
I'm a bit scared of this change.

I'm pretty sure the old text does nothing, but I don't know if the new text does anything.

Maybe we should delete it?

```suggestion
```

Or do:

```suggestion
%options easy_keword_rules // cspell: disable-line
```

It was originally `%options easy_keyword_rules` before https://github.com/mermaid-js/mermaid/commit/6077ba54054eed165052d73fe342b6ad3bf04e8c